brake/backends/cachebe.py

```diff
@@ -71,7 +73,7 @@ def limit(self, func_name, request,
             if ':ip:' in counter:
                 ratelimited_by = 'ip'
 
-            if counters[counter] > count:
+            if counters[counter][0] > count:
```

Comment:
I don't really feel strongly about it, I'm just trying to understand what you're wary about. I think we should check regardless and replace the value if it's not a tuple, as that means that the library was just upgraded and there is old, integer-only data in the cache. I'll commit that soon.